harness/determined/cli/job.py

```diff
@@ -152,17 +137,32 @@ def check_is_priority(config: dict, resource_pool: str) -> bool:
     return True
 
 
-def replace_keys(operations: dict) -> None:
+def validate_operation_args(operation: str) -> dict:
     replacements = {
         "resource-pool": "resource_pool",
         "ahead-of": "ahead_of",
         "behind-of": "behind_of",
     }
-    for k, v in replacements.items():
-        if k in operations:
-            operations[v] = operations[k]
-            del operations[k]
+    args = {}
+    values = operation.split(".")
+    if len(values) != 2:
+        raise ValueError(
+            f"Invalid operation specified for job {values[0]}. "
```

Comment:
this says invalid operation but it doesn't actually check for the validity of the operation name?